bspump/declarative/expression/string/regex.py

```diff
@@ -84,30 +86,44 @@ def __init__(self, app, *, arg_regex, arg_what, arg_items=None, arg_miss=None, a
 
 		# TODO: Regex flags
 
+	def set(self, key, value):
```

Comment:
?

Reply:
@ateska The same as above. Expressions in lists or dicts do not pass optimization.